client/src/docs-ui/chat-button/chat-button.style.ts

```diff
@@ -25,6 +25,6 @@ export const gitterChatStyle = css`
   }
 
   a {
-    color: var(--font-color);
+    color: var(--color-white);
```

Comment:
Let's make this a font variable like `--font-color-contrast` instead of using `--color-white`